web-common/src/features/dashboards/state-managers/selectors/pivot.ts

```diff
@@ -1,13 +1,14 @@
+import { filteredBasicMeasures } from "@rilldata/web-common/features/dashboards/state-managers/selectors/measures";
 import type { DashboardDataSources } from "./types";
 import { PivotChipType } from "../../pivot/types";
 
 export const pivotSelectors = {
   showPivot: ({ dashboard }: DashboardDataSources) => dashboard.pivot.active,
   rows: ({ dashboard }: DashboardDataSources) => dashboard.pivot.rows,
   columns: ({ dashboard }: DashboardDataSources) => dashboard.pivot.columns,
-  measures: ({ metricsSpecQueryResult, dashboard }: DashboardDataSources) => {
-    const measures = metricsSpecQueryResult.data?.measures ?? [];
-    const columns = dashboard.pivot.columns;
+  measures: (dashData: DashboardDataSources) => {
```

Comment:
**Issue**: The refactoring of the 'measures' selector function changes the function signature, which could lead to issues if other parts of the application rely on the original signature and expected behaviors. <br> **Fix**: Ensure that all references to the 'measures' selector across the application are updated to pass the correct type of argument, or revert to the original function signature to maintain compatibility. <br> **Code Suggestion**: 
 ```
 - measures: ({ metricsSpecQueryResult, dashboard }: DashboardDataSources) => {
 + measures: (dashData: DashboardDataSources) => {
 ```